test/pipelines/tfjs-mobilenet-classification.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
You should patch this on https://github.com/alibaba/pipcook/blob/master/test/pipelines/mobilenet-classification.json :)